server.js

```diff
@@ -0,0 +1,5 @@
+const app = require('./app')
+
+app.listen(8000, function(){
+    console.log('Listening port 8000')
+})
```

Comment:
Nice extraction!
I might suggest something like `listen.js` over `server.js` for this